src/System.Private.CoreLib/src/System/ArgIterator.cs

```diff
@@ -132,54 +131,5 @@ public override bool Equals(Object o)
         {
             throw new NotSupportedException(SR.NotSupported_NYI);
         }
-#else
-        public ArgIterator(RuntimeArgumentHandle arglist)
-        {
```

Comment:
You should keep this throwing implementation for Unix until/if the varargs are actually implemented for Unix

Reply:
Is there a platform specific define that can be used here?

Reply:
`PLATFORM_UNIX` or `PLATFORM_WINDOWS`